source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -782,6 +789,9 @@ thread SHOULD
   Timeout: Background Connection Pooling
   <../client-side-operations-timeout/client-side-operations-timeout.rst#background-connection-pooling>`__.
 
+A pool SHOULD allow ability to force run next maintenance iteration to remove perished connections including "in use" connections. 
+In this case, A pool MAY skip populating connections.
```

Comment:
not sure whether this step is necessary? It looks like `minPoolSize` can be handled as before even though they will be available only after pool will become `ready`

Reply:
1. Maybe "In this case, A pool MAY skip populating connections." can be omitted, as it's part of pausable behaviour: "Connections are not created in the background to satisfy minPoolSize"
2. "force run next maintenance iteration" ==> "Schedule the next Background Thread Run to run as soon as possible" or similar

Reply:
done